doc/api/errors.md

```diff
@@ -3180,6 +3180,9 @@ Creation of a [`zlib`][] object failed due to incorrect configuration.
 
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/54125
+    description: Corrected the description of the max header size to `maxHeaderSize`.
```

Comment:
```suggestion
```

I might be wrong, but I think this is only for changes to the API, not the docs

Reply:
I thought the same thing but the comment below says

```
description: Max header size in `http_parser` was set to 8 KiB.
```

which is misleading.

Reply:
The commit for that one changes the API (186035243fad247e3955fa0c202987cae99e82db)

Reply:
@RedYetiDev @lpinca 
The PR changed from 8 KiB to 16 KiB appears to be [here](https://github.com/nodejs/node/pull/32520).

The following change log is left in other than `errors.md`, is such a description also unnecessary?
https://github.com/nodejs/node/blob/e4f61de14f8cfb83f1ce0ad1597b86278cd5f5f1/doc/api/cli.md?plain=1#L1554-L1557

Reply:
As said above, there was no particular change log description in `http.md`.

https://github.com/nodejs/node/blob/e4f61de14f8cfb83f1ce0ad1597b86278cd5f5f1/doc/api/http.md?plain=1#L3751-L3763

I thought the description might not be necessary this time, so I'm going to remove it

Reply:
fixed https://github.com/nodejs/node/pull/54125/commits/944900b1db205bb9dcac23d55e427d636455c8e8

Reply:
Looks good!